tools/isopack.js

```diff
@@ -1100,18 +1100,64 @@ _.extend(Isopack.prototype, {
         !f.match(/^examples\/unfinished/);
     });
 
-    var gitSha = files.runGitInCheckout('rev-parse', 'HEAD');
-
+    // Regexes matching paths to transpile using babel
+    var transpileRegexes = [
+      /^tools\/[^\/]+\.js$/, // General tools files
+      // We don't support running self-test from an install anymore
+    ];
+
+    // Split pathsToCopy into two arrays - one of files that should be copied
+    // directly, and one of files that should be transpiled with Babel
+    var pathsToTranspile = [];
+    var pathsToCopyStraight = [];
+    pathsToCopy.forEach((path) => {
+      var shouldTranspile =
+        _.some(transpileRegexes, (regex) => path.match(regex));
+
+      if (shouldTranspile) {
+        pathsToTranspile.push(path);
+      } else {
+        pathsToCopyStraight.push(path);
+      }
+    });
 
+    // Set up builder to write to the correct directory
     var toolPath = 'mt-' + archinfo.host();
     builder = builder.enter(toolPath);
+
+    // Transpile the files we selected
+    var babel = require("babel");
```

Comment:
Are we not attempting to provide Babel as a Meteor package and load it as an isopacket?